Modules/Core/Common/include/itkImportImageContainer.h

```diff
@@ -162,12 +161,11 @@ class ITK_TEMPLATE_EXPORT ImportImageContainer : public Object
   PrintSelf(std::ostream & os, Indent indent) const override;
 
   /**
-   * Allocates elements of the array.  If UseDefaultConstructor is true, then
-   * the default constructor is used to initialize each element.  POD date types
-   * initialize to zero.
+   * Allocates elements of the array.  If UseInitValue is true, then POD
+   * types will be zero-initialized.
    */
   virtual TElement *
-  AllocateElements(ElementIdentifier size, bool UseDefaultConstructor = false) const;
+  AllocateElements(ElementIdentifier size, bool UseInitValue = false) const;
```

Comment:
Maybe `bool UseValueInitialization`? Or simply `bool ValueInitialize`? If we really want to be precise, I think we cannot avoid the technical C++ term "value-initialization" anyway. 🤷